setup/environment36.yml

```diff
@@ -65,22 +65,21 @@ dependencies:
 - ipywidgets=6.0.0=py36_0
 - isort=4.2.5=py36_0
 - itsdangerous=0.24=py36_0
-- jbig=2.1=0
 - jdcal=1.3=py36_0
 - jedi=0.10.2=py36_2
 - jinja2=2.9.6=py36_0
-- jpeg=9b=0
+- jpeg
 - jsonschema=2.6.0=py36_0
 - jupyter=1.0.0=py36_3
 - jupyter_client=5.0.1=py36_0
 - jupyter_console=5.1.0=py36_0
 - jupyter_core=4.3.0=py36_0
 - lazy-object-proxy=1.2.2=py36_0
-- libiconv=1.14=0
-- libpng=1.6.27=0
-- libtiff=4.0.6=3
-- libxml2=2.9.4=0
-- libxslt=1.1.29=0
+- libiconv=1.14
+- libpng=1.6.27
+- libtiff=4.0.6
+- libxml2
```

Comment:
Try `libxml2=2.9.4`
Again, there appears to be a Windows version for this
https://anaconda.org/anaconda/libxml2/files